### PR TITLE
Cleanup mtarget views

### PR DESCRIPTION
**Pull request by frzyc**
## Describe your changes

Clean up the main mtarget modal, eliminate the accordions, instead opt to use cards. 
Clean up the import/export flow.

## Issue or discord link

- https://github.com/frzyc/genshin-optimizer/issues/1834

## Testing/validation

Main mtarget management view
image

Mtarget editor
image

Mtarget import modal
image

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

**Comment by nguyentvan7** [comment]
We are 3 modals deep, do you think this is too many? genuine question

File: apps/frontend/src/app/PageTeam/CharacterDisplay/CustomMultiTarget/CustomMultiTargetCard.tsx [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+    (target: string[], multi?: number) => {
+      onClose()
+      setTarget(target, multi)
+    },
+    [onClose, setTarget]
+  )
+
+  return (
+    <>
+      <Button
```
Comment (nguyentvan7) [inline apps/frontend/src/app/PageTeam/CharacterDisplay/CustomMultiTarget/CustomMultiTargetCard.tsx:309]: Should we update any new buttons to be outlined to save some migration effort later?

File: apps/frontend/src/app/PageTeam/CharacterDisplay/CustomMultiTarget/CustomMultiTargetCard.tsx [hunk 2]
```diff
@@ -0,0 +1,326 @@
+import { useBoolState } from '@genshin-optimizer/common/react-util'
```
Comment (nguyentvan7) [inline apps/frontend/src/app/PageTeam/CharacterDisplay/CustomMultiTarget/CustomMultiTargetCard.tsx:1]: We lost the ability to re-order multi-opt, but I assume that is coming back via DnD?

File: apps/frontend/src/app/PageTeam/CharacterDisplay/CustomMultiTarget/CustomMultiTargetCard.tsx (hunk 1 above)
Comment (frzyc) [reply apps/frontend/src/app/PageTeam/CharacterDisplay/CustomMultiTarget/CustomMultiTargetCard.tsx:309]: Probably will get addressed later, need to see how the final layout look like.

File: apps/frontend/src/app/PageTeam/CharacterDisplay/CustomMultiTarget/CustomMultiTargetCard.tsx (hunk 2 above)
Comment (frzyc) [reply apps/frontend/src/app/PageTeam/CharacterDisplay/CustomMultiTarget/CustomMultiTargetCard.tsx:1]: possibly

**Comment by frzyc** [comment]
> We are 3 modals deep, do you think this is too many? genuine question

probably, through I still like this better than cramming all the mtarget UI together with collapse and all those buttons.

**Review by nguyentvan7 — APPROVED** [review]

**Pull request merged**